api-definitions/db_positions/api.yaml

```diff
@@ -0,0 +1,292 @@
+swagger: "2.0"
+info:
+  version: 1.0.1
```

Comment:
What's the point of a patch number in the spec version?

Reply:
Nobody, it's a change test, i'll rollback